info.yml

```diff
@@ -92,4 +92,5 @@ requires_engine_version:
 # the frameworks required to run this app
 frameworks:
     - {"name": "tk-framework-shotgunutils", "version": "v5.x.x", "minimum_version": "v5.2.0"}
+     # TODO: Update required qtwidgets once the new version is released
```

Comment:
TODO blocker

Reply:
Yeah I just re-added the DO NOT MERGE label, I can't do anything about it until the other framework is released.